requirements.txt

```diff
@@ -126,9 +126,7 @@ python-dateutil==2.8.2
 python3-openid==3.2.0
     # via social-auth-core
 pytz==2022.7.1
-    # via
-    #   django
-    #   django-notifications-hq
+    # via django-notifications-hq
```

Comment:
side note, django-notifications-hq is documented to only work with up to django 4.1 but looks like it works with 4.2 too